### PR TITLE
Lots of updates

### PR DESCRIPTION
**Pull request by Mycroft-Studios**

### CODE REVIEW

**Comment by slyjordi** [comment]
👌

**Review by slyjordi — APPROVED** [review]

**Review by slyjordi — APPROVED** [review]

**Review by roadtosixx — APPROVED** [review]

**Review by jrence — APPROVED** [review]

File: [esx_addons]/esx_clotheshop/config.lua [hunk 1]
```diff
@@ -1,13 +1,22 @@
 Config = {}
 Config.Locale = GetConvar('esx:locale', 'en')
 
-Config.Price = 250
+Config.Price = math.random(200, 300)
```
Comment (mcNuggets1) [inline [esx_addons]/esx_clotheshop/config.lua:4]: The fuck, you dont know how math random works, do you?

**Comment by slyjordi** [comment]
merge this 😥😥😥😥😥😥😥😥😥😥

**Comment by jbdevic** [comment]
@Mycroft-Studios please fix conflicts

**Comment by TheFantomas** [comment]
I'm going to take a look at it today and see if this gets my vote or not :D I've been testing it for a while now and sometimes I had a not-so-nice problem :D And that's with things related to death or clothes - **it didn't have time to update the ESX.PlayerData.Ped values and that's the end of it, so I wouldn't replace it everywhere :D**

So before I wrote this comment I had a quick look :D And it looks like a nice mini optimization improvement.

**Comment by mcNuggets1** [comment]
Isn't it slower to access the global ESX table every frame instead of calling PlayerPedId?

**Comment by TheFantomas** [comment]
> Isn't it slower to access the global ESX table every frame instead of calling PlayerPedId?

No, practically not. It's already a stored value so no.

**Comment by Mycroft-Studios** [comment]
closing, going to do these as seperate PRs

**Pull request closed without merging**